packages/react/src/components/DotcomShell/__stories__/DotcomShell.stories.js

```diff
@@ -34,7 +34,9 @@ storiesOf('Dotcom Shell', module)
         )}
         footerType={select('Footer', footer, footer.default)}
         hasProfile={boolean('Has profile', true)}
-        hasSearch={boolean('Has search', true)}>
+        hasSearch={boolean('Has search', true)}
+        title={text('Title', 'Stock Charts')}
+        eyebrowLink={text('Eyebrow link', '#')}>
```

Comment:
@emerson-pereira since the L1 is still behind a feature flag, can you conditionally add these props in the story with the feature flag?